agdb_api/typescript/src/query_builder.ts

```diff
@@ -308,6 +308,41 @@ class InsertIndexBuilder {
     }
 }
 
+function convertToDbValue(value: any): Components.Schemas.DbValue {
+    if (typeof value === 'string') {
+        // String type
+        return { String: value };
+    } else if (typeof value === 'boolean') {
+        return { String: value ? 'true' : 'false' };
```

Comment:
The conversion of a boolean to an I64 was not considered because the reverse conversion would lose the information about whether it was originally a boolean type or a number.

Reply:
Is that different to the `String` type? `String` value "true" or "false" could have originally be of type `String` instead of `boolean` but I agree that it would be somewhat less ambiguous. Okay, just keep it as `String` and I will see what can be done. They will be small value (<15bytes anyway).

I want to support booleans a bit better (#1088). The problem with them is that the minimum value size is 16 bytes so originally I have left it ambiguous on purpose letting users choose how they want to represent them.